File: mobile/src/main/java/com/aw/avi/allutilsmethodsapp/ui/activity/KotlinMainActivity.java
package com.aw.avi.allutilsmethodsapp.ui.activity;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.aw.avi.allutilsmethodsapp.R;
import com.aw.avi.allutilsmethodsapp.ui.activity.kotlin.KotlinBasicActivity;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class KotlinMainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_kotlin_main);
        ButterKnife.bind(this);


    }

    @OnClick({R.id.bt_kotlin_first, R.id.bt_kotlin_logics})
    public void onViewClicked(View view) {
        switch (view.getId()) {
            case R.id.bt_kotlin_first:
                startActivity(new Intent(this, KotlinBasicActivity.class));
                break;
            case R.id.bt_kotlin_logics:
                break;
        }
    }
}

File: mobile/src/main/java/com/aw/avi/allutilsmethodsapp/ui/activity/KotlinMainActivity.kt
package com.aw.avi.allutilsmethodsapp.ui.activity

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.aw.avi.allutilsmethodsapp.R
import com.aw.avi.allutilsmethodsapp.kotlin.ui.AdsInKotlinActivity
import com.aw.avi.allutilsmethodsapp.kotlin.ui.KotlinBasicActivity
import kotlinx.android.synthetic.main.activity_kotlin_main.*

class KotlinMainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_kotlin_main)

        bt_kotlin_first.setOnClickListener {
            startActivity(Intent(this, KotlinBasicActivity::class.java))
        }

        bt_kotlin_logics.setOnClickListener {
            startActivity(Intent(this, KotlinBasicActivity::class.java))
        }

        bt_ads_in_kotlin.setOnClickListener {
            startActivity(Intent(this, AdsInKotlinActivity::class.java))
        }




    }

//    @OnClick(R.id.bt_kotlin_first, R.id.bt_kotlin_logics)
//    fun onViewClicked(view: View) {
//        when (view.id) {
//            R.id.bt_kotlin_first -> startActivity(Intent(this, KotlinBasicActivity::class.java))
//            R.id.bt_kotlin_logics -> {
//            }
//        }
//    }
}
